lib/ColField/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { propTypes, defaultProps } from './common';
+import ColFieldHeader from './ColFieldHeader';
+import ColFieldLabel from './ColFieldLabel';
+import ColFieldBody from './ColFieldBody';
+import ColFieldFooter from './ColFieldFooter';
+import ColFieldInstructions from './ColFieldInstructions';
+
+function ColField({ children, className, ...props }) {
```

Comment:
💚 I think we chatted about not using the `/index.js` filename, as a `ColField/ColField.js` is easier for IDEs to find.

We may not have gone down to the wire and discussed this yet though, so I'll raise it as a FE retro point!

Reply:
This is something that was raised! I just forgot (I think its because we have way more offending index files in GUI)